sonar/modules/documents/views.py

```diff
@@ -289,6 +289,35 @@ def is_file_restricted(file, record):
     :param record: Current record
     :returns object containing result and possibly embargo date
     """
+
+    def is_restricted_by_scope(file):
+        """File is restricted by scope (internal, rero or institution).
+
+        :param file: File object.
+        """
+        # File is restricted by internal IPs
+        if file['restricted'] in ['rero', 'internal']:
```

Comment:
maybe the 'rero' string is too restrictive to be used in other projects (I'm already thinking about using Sonar@UCLouvain). Maybe possible to use a more generic name ? "consortium", "association" ?

Reply:
I just kept "internal", "rero" is not necessary.